packages/core/content-manager/admin/src/preview/components/PreviewContent.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+
+import { Page } from '@strapi/admin/strapi-admin';
+
+import { usePreviewContext } from '../pages/Preview';
+
+const PreviewContent = () => {
+  const previewUrl = usePreviewContext('PreviewIframe', (state) => state.url);
+
+  if (!previewUrl) {
+    return <Page.Loading />;
+  }
+
+  return (
+    <iframe
+      src={previewUrl}
+      title="Content Preview"
+      style={{
+        width: '100%',
+        height: '100%',
+        border: 'none',
+      }}
+    />
```

Comment:
- the title needs to be translated. Not sure if there's a json key that can be reused
- we try to use the design system as much as possible for styles. Could you try implementing this with `<Box tag="iframe"`?